### PR TITLE
Deprecate the networkExplorer option

### PR DESCRIPTION
**Pull request by dmitryax**
Add instruction on how to replace networkExplorer with the upstream OpenTelemetry eBPF Helm Chart.

### CODE REVIEW

**Review by atoulme — APPROVED** [review]

File: docs/advanced-configuration.md [hunk 1]
```diff
@@ -780,3 +780,35 @@ agent:
   * Cluster receiver is a 1-replica deployment of Open-temlemetry collector.
   * As any available node can be selected by the Kubernetes control plane to run the cluster receiver pod (unless we explicitly specify the `clusterReceiver.nodeSelector` to pin the pod to a specific node), `hostPath` or `local` volume mounts wouldn't work for such envrionments.
   * Data Persistence is currently not applicable to the k8s cluster metrics and k8s events.
+
+### Using OpenTelemetry eBPF helm chart with Splunk OpenTelemetry Collector for Kubernetes
```
Comment (aunshc) [inline docs/advanced-configuration.md:784]: Question: Is there a set of generic instructions that customers will be able to leverage if they are using the upstream helm chart instead?

File: UPGRADING.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Upgrade guidelines
 
+## 0.87.0 to 0.88.0
+
+The `networkExplorer` option is deprecated now. Please use the upstream OpenTelemetry eBPF Helm chart to collect
```
Comment (jinja2) [inline UPGRADING.md:5]: Should we add more instructions about how to switch? Is it recommended for users to first disable the nw explorer from Splunk distro release and then install the new release? I assume we might not want both the running simultaneously.

Comment (dmitryax) [reply UPGRADING.md:5]: Good point. updated 👍

File: docs/advanced-configuration.md (hunk 1 above)
Comment (dmitryax) [reply docs/advanced-configuration.md:784]: discussed offline

**Pull request merged**